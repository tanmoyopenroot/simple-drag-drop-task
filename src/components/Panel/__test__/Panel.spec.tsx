import * as React from 'react';
import { shallow } from 'enzyme';

import {
  Panel,
  IPanelProps,
} from '../Panel';

describe('Testing <Panel />', () => {
  const props: IPanelProps = {
    backgroundColor: 'gray',
  };
  const wrapper = shallow<IPanelProps>(
    <Panel {...props}>Some Text</Panel>,
  );

  it('should be able to render with props', () => {
    expect(wrapper.length).toEqual(1);
    expect(wrapper.children().length).toBe(1);
    expect(wrapper.props().backgroundColor).toEqual(props.backgroundColor);
    expect(wrapper).toMatchSnapshot();
  });
});
