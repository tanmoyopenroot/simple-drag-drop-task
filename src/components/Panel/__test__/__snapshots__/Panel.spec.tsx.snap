// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Panel /> should be able to render with props 1`] = `ShallowWrapper {}`;
